build/VStudio/installer/Product.wxs

```diff
@@ -53,8 +54,8 @@
                     Key="[P.RegistryKey]"
                     Name="InstallDir"
                     Type="string"
-                    Value="[INSTALLDIR]"
-                    KeyPath="yes" />

```

Comment:
Is there a reason that you remove the `KeyPath="yes"` option here?

Reply:
Actually the build fails. Seems like the `dep:Provides` element provides the `KeyPath`. So I understand your change.